redis/Cargo.toml

```diff
@@ -65,11 +65,11 @@ tokio-native-tls = { version = "0.3", optional = true }
 async-native-tls = { version = "0.5", optional = true }
 
 # Only needed for rustls
-rustls = { version = "0.22", optional = true }
+rustls = { version = "0.23", optional = true, default-features = false }
 webpki-roots = { version = "0.26", optional = true }
 rustls-native-certs = { version = "0.7", optional = true }
-tokio-rustls = { version = "0.25", optional = true }
-futures-rustls = { version = "0.25", optional = true }
+tokio-rustls = { version = "0.26", optional = true, default-features = false }
+futures-rustls = { version = "0.26", optional = true }
```

Comment:
Sorry, I missed that there's also a futures-rustls dependency here -- that also needs the `default-features = false`.